addon/components/bs-modal.js

```diff
@@ -193,6 +193,20 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+    /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using 'true' and
```

Comment:
`true` was using quotes instead of backticks and therefore won't be formatted as code.

```suggestion
   *  apply the `modal-fullscreen' class when using `true` and
```

Reply:
OK, but in this case, some of the other comments have also quotes !
So we need to update all of these ?

Reply:
It is okay to only address the new API docs added by this pull request here. I would recommend to fix other issues in separate pull requests to keep focus.

I just noticed that at the end of `modal-fullscreen` it is also a single quote and not a backtick. Here is an updated suggestion.

```suggestion
   *  apply the `modal-fullscreen` class when using `true` and
```

Reply:
Fixed